### DISCUSSION TITLE
right/left to move to the next or previous entry is not working

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#16486] [How to]
As per below info, the shortcuts introduced seems not working. Any quick fix? Essential would be navigation between records (detail and summary view) 
https://github.com/YetiForceCompany/YetiForceCRM/discussions/15466

**Comment by mariuszkrzaczkowski** [comment]
the current abbreviation list is here, we're talking about the latest stable release

https://yetiforce.com/en/knowledge-base/documentation/user-documentation/item/keyboard-shortcuts
